CalendarApp/NUnitTestProject/BookingServiceTests.DeleteTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Linq;
+using BookingRoom.Models.GoogleEvent;
+using Google.Apis.Calendar.v3;
+using NUnit.Core;
+using NUnit.Framework;
+using BookingRoom.Helpers;
+using Microsoft.Practices.Unity;
+using GoogleCalendarService;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests : BaseIntegrationTest
+    {
+        private IUnityContainer _unityContainer;
+        private CalendarService _calendarService;
+        private BookingService _meetingBooking;
+
+        [TestFixtureSetUp]
+        public void Initialize()
+        {
+            _unityContainer = Dependency.UnityConfig.GetUnityContainer();
+            _calendarService = _unityContainer.Resolve<CalendarService>();
+            _meetingBooking = _unityContainer.Resolve<BookingService>();
+        }
+
+        [Test]
+        public void BookingService_DeleteEvent_WrongCalendarIdThrowException()
+        {
+            var eventId = "abqq3960anfn5cmi1242usco";
+            var calendarId = string.Empty;//error
```

Comment:
If you are testing that single property should throw than other properties have to be valid. Otherwise it is not guarantied that exception is thrown because of that specific property